### PR TITLE
Ft brand banner

### PR DESCRIPTION
**Pull request by EliusCaleb**
# 322  Create Elewa Education brand banner

Description
The banner has the following

1. Brand Logo
2. Brand Name
3. Brands Description
4. Button

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Screenshot 
image

# How Has This Been Tested?
Its renders on the main browser

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: libs/pages/elewa/brands/src/lib/brands.routing.ts [hunk 1]
```diff
@@ -14,4 +14,5 @@ export const ELEWA_BRANDS_ROUTES: Route[] = [
   imports: [RouterModule.forChild(ELEWA_BRANDS_ROUTES)],
   exports: [RouterModule]
 })
-export class BrandsRoutingModule { }
\ No newline at end of file
+export class BrandsRoutingModule { }
+
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/brands/src/lib/brands.routing.ts]: This change is not necessary

File: libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts [hunk 2]
```diff
@@ -9,9 +9,14 @@ import { ElewaBrandsOpportunitiesComponent } from './components/elewa-brands-opp
 import { ElewaBrandsPageComponent } from './pages/elewa-brands-page/elewa-brands-page/elewa-brands-page.component';
 
 import { BrandsRoutingModule } from './brands.routing';
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts:11]: router should be last import

File: libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts [hunk 3]
```diff
@@ -9,9 +9,14 @@ import { ElewaBrandsOpportunitiesComponent } from './components/elewa-brands-opp
 import { ElewaBrandsPageComponent } from './pages/elewa-brands-page/elewa-brands-page/elewa-brands-page.component';
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts:9]: Pages imported after components

File: libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts [hunk 4]
```diff
@@ -9,9 +9,14 @@ import { ElewaBrandsOpportunitiesComponent } from './components/elewa-brands-opp
...
   imports: [CommonModule, LayoutModule, BannersModule, BrandsRoutingModule],
-  declarations: [ElewaBrandsPageComponent, ElewaBrandsOpportunitiesComponent],
+  declarations: [
+    ElewaBrandsPageComponent,
+    ElewaBrandsOpportunitiesComponent,
+    ElewaBrandsElewaEducationComponent,
+  ],
 })
-export class BrandsModule {}
+export class BrandsModule {}
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts:22]: Not necessary change

File: libs/pages/elewa/brands/src/lib/pages/elewa-brands-page/elewa-brands-page/elewa-brands-page.component.spec.ts [hunk 5]
```diff
@@ -20,3 +20,4 @@ describe('ElewaBrandsPageComponent', () => {
     expect(component).toBeTruthy();
   });
 });
+
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/brands/src/lib/pages/elewa-brands-page/elewa-brands-page/elewa-brands-page.component.spec.ts:23]: Not necessary change

File: libs/pages/elewa/brands/src/lib/pages/elewa-brands-page/elewa-brands-page/elewa-brands-page.component.ts [hunk 6]
```diff
@@ -5,4 +5,4 @@ import { Component } from '@angular/core';
   templateUrl: './elewa-brands-page.component.html',
   styleUrls: ['./elewa-brands-page.component.scss'],
 })
-export class ElewaBrandsPageComponent {}
+export class ElewaBrandsPageComponent {}
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/brands/src/lib/pages/elewa-brands-page/elewa-brands-page/elewa-brands-page.component.ts:8]: Not necessary change

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

File: libs/pages/elewa/brands/src/lib/brands.routing.ts (hunk 1 above)
Comment (EliusCaleb) [reply libs/pages/elewa/brands/src/lib/brands.routing.ts]: It was an issue to create the  brands lib component

**Review by IanOdhiambo9 — APPROVED** [review]

**Pull request merged**